runtime/src/snapshot_utils.rs

```diff
@@ -855,6 +839,77 @@ pub fn bank_from_snapshot_archives(
             None
         };
 
+    Ok((unarchived_full_snapshot, unarchived_incremental_snapshot))
+}
+
+pub fn bank_fields_from_snapshot_archives(
+    // Pass in path to the directory containing the snapshot archive
+    bank_snapshots_dir: impl AsRef<Path>,
+) -> Result<BankFieldsToDeserialize> {
+    let full_snapshot_archive_info = get_highest_full_snapshot_archive_info(&bank_snapshots_dir)
+        .ok_or(SnapshotError::NoSnapshotArchives)?;
+
+    let incremental_snapshot_archive_info = get_highest_incremental_snapshot_archive_info(
+        &bank_snapshots_dir,
+        full_snapshot_archive_info.slot(),
+    );
+
+    let temp_dir = tempfile::Builder::new()
+        .prefix("dummy-accounts-path")
+        .tempdir_in(&bank_snapshots_dir)?;
```

Comment:
Does this temp dir need to be in the bank snapshots dir? My initial thought is 'no', so a `tempfile::tempdir();` should be OK here, yes?

Reply:
done!